app/policies/vye_policy.rb

```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+
+VyePolicy = Struct.new(:user, :user_info) do
+  def access?
+    return true if user.present?
```

Comment:
Small nitpick but you can just do `user.present?` without an extra `return true` and `false` here

Reply:
This change has been made